docs/glossary.rst

```diff
@@ -0,0 +1,54 @@
+.. -*- coding: utf-8-with-signature -*-
+
+=======================
+Glossary of Tahoe terms
+=======================
+
+
+.. glossary::
+
+    cap
+    capability
+        dircap, filecap, write cap, read cap, verify cap; all these refer to a capability or privilege associated with a key.
+        The word "Capability" comes from "Object Capability" or OCap theory, in the manner of `the E language <http://erights.org/elib/capability/ode/ode-capabilities.html>`_.
```

Comment:
Oh, weird, I tried "edit file" to add more detail -- I thought it would show up as one of those "suggested changes". How _does_ one do "suggested changes"?

Reply:
Hi @meejah , in markdown you can do:

```suggestion

at the top of the code block and github will read that block as the patch.

Reply:
ooops, it read it too literally, what I meant is that in the comment field, start a new line with 3 backticks and `suggestion` on a line by itself.
Then in the code block, put the contents of change.